components/shared/history/HistoryBrowser.vue

```diff
@@ -2,9 +2,7 @@
   <div v-show="history && history.length" class="history-browser">
     <b-dropdown
       aria-role="list"
-      scrollable
-      position="is-bottom-left"
-      max-height="400px">
```

Comment:
Why did you remove these lines?
https://buefy.org/documentation/dropdown/#scrollable